install/gcp/installer/forseti_server_installer.py

```diff
@@ -101,16 +106,20 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
             instance_name = 'forseti-{}-vm-{}'.format(
                 self.config.installation_type,
                 self.config.identifier)
-            self.wait_until_vm_initialized(instance_name)
 
             # Create firewall rules.
             self.create_firewall_rules()
+            self.wait_until_vm_initialized(instance_name)
+
+            # Delete firewall rules.
+            self.delete_firewall_rules()
 
         return success, deployment_name
 
     def create_firewall_rules(self):
         """Create firewall rules for Forseti server instance."""
-        # Rule to block out all the ingress traffic.
+        # This rule overrides the implied deny for ingress
+        # because it has higher priority and is specific to service account.
```

Comment:
I want to add to this comment, the "why", which is to make it harder to override.  Can you please add this?  Here and in the client?